turbinia/turbiniactl.py

```diff
@@ -36,7 +36,7 @@
 logger.setup()
 
 
-if __name__ == '__main__':
+def main():
```

Comment:
Can you also add something like the following at the bottom so the script will run without installing it?
```
if __name__ == '__main__':
    main()
```